crates/bevy_pbr/src/ssr/mod.rs

```diff
@@ -233,15 +234,33 @@ impl Plugin for ScreenSpaceReflectionsPlugin {
 
         render_app
             .init_resource::<ScreenSpaceReflectionsPipeline>()
-            .init_resource::<SpecializedRenderPipelines<ScreenSpaceReflectionsPipeline>>()
-            .add_render_graph_edges(
+            .init_resource::<SpecializedRenderPipelines<ScreenSpaceReflectionsPipeline>>();
+
+        // only reference the default deferred lighting pass
+        // if it has been added
+        let has_default_deferred_lighting_pass = render_app
+            .world_mut()
+            .get_resource_mut::<RenderGraph>()
+            .unwrap()
```

Comment:
This should essentially never be reached, but I'd prefer if this used expect with an error message. Alternatively, just use `.resource_mut` which should already panic with a decent error message

Reply:
Used `resource_mut`, I agree that that is the more idiomatic approach